.ci/scripts/test_int_cslc_s1.sh

```diff
@@ -0,0 +1,254 @@
+      shift
+      shift
+      ;;
+    *)
+      echo "Unknown argument $1, ignoring..."
+      shift
+      ;;
+  esac
+done
+
+echo '
+================================================
+Integration Testing CSLC_S1 PGE docker image...
+================================================
+'
+
+PGE_NAME="cslc_s1"
+PGE_IMAGE="opera_pge/${PGE_NAME}"
+
+TEST_RESULTS_REL_DIR="test_results"
+
+# defaults, test data and runconfig files should be updated as-needed to use
+# the latest available as defaults for use with the Jenkins pipeline call
+# TESTDATA should be the name of the test data archive in s3://operasds-dev-pge/cslc_s1/
+# RUNCONFIG should be the name of the runconfig in s3://operasds-dev-pge/cslc_s1/
+[ -z "${WORKSPACE}" ] && WORKSPACE=$(realpath $(dirname $(realpath $0))/../..)
+[ -z "${TESTDATA}" ] && TESTDATA="delivery_cslc_s1_interface_0.1.zip"
+[ -z "${RUNCONFIG}" ] && RUNCONFIG="cslc_s1.yaml"
+
+TEST_RESULTS_DIR="${WORKSPACE}/${TEST_RESULTS_REL_DIR}/${PGE_NAME}"
+
+echo "Test results output directory: ${TEST_RESULTS_DIR}"
+mkdir --parents ${TEST_RESULTS_DIR}
+chmod -R 775 ${TEST_RESULTS_DIR}
+RESULTS_FILE="${TEST_RESULTS_DIR}/test_int_${PGE_NAME}_results.html"
+
+# Create a temporary directory to allow Jenkins to write to it and avoid collisions
+# with other users
+local_dir=$(mktemp -dp /tmp)
+chmod 775 $local_dir
+cd $local_dir
+
+local_testdata_archive=${local_dir}/${TESTDATA}
+local_runconfig=${local_dir}/${RUNCONFIG}
+
+results_html_init="<html><b>${PGE_NAME} product comparison results</b><p> \
+    <style>* {font-family: sans-serif;} \
+    table {border-collapse: collapse;} \
+    th,td {padding: 4px 6px; border: thin solid white} \
+    tr:nth-child(even) {background-color: whitesmoke;} \
+    </style><table>"
+
+echo $results_html_init > $RESULTS_FILE
+
+# Configure a trap to set permissions on exit regardless of whether the testing succeeds
+function cleanup {
+
+    echo "</table></html>" >> $RESULTS_FILE
+    DOCKER_RUN="docker run --rm -u $UID:$(id -g)"
+
+    echo "Cleaning up before exit. Setting permissions for output files and directories."
+    ${DOCKER_RUN} -v ${local_dir}:${local_dir} --entrypoint /usr/bin/find ${PGE_IMAGE}:${PGE_TAG} ${local_dir} -type d -exec chmod 775 {} +
+    ${DOCKER_RUN} -v ${local_dir}:${local_dir} --entrypoint /usr/bin/find ${PGE_IMAGE}:${PGE_TAG} ${local_dir} -type f -exec chmod 664 {} +
+    cd /tmp
+    rm -rf ${local_dir}
+}
+
+trap cleanup EXIT
+
+# Pull in test data and runconfig from S3
+echo "Downloading test data from s3://operasds-dev-pge/${PGE_NAME}/${TESTDATA} to $local_testdata_archive"
+aws s3 cp s3://operasds-dev-pge/${PGE_NAME}/${TESTDATA} $local_testdata_archive
+echo "Downloading runconfig from s3://operasds-dev-pge/${PGE_NAME}/${RUNCONFIG} to $local_runconfig"
+aws s3 cp s3://operasds-dev-pge/${PGE_NAME}/${RUNCONFIG} $local_runconfig
+
+# Pull in validation script from S3. This utility doesn't appear to be source-controlled
+# so we have cached the delivery version in S3.
```

Comment:
Just an FYI, ADT has finally merged this script into the CSLC SAS repo, so for future deliveries we should be able to count on it being included within the container